### ISSUE TITLE
Multiplication of road sections in a restriction relation

### ISSUE THREAD

**Issue by maro-21** [Zaczero/osm-relatify#81]
I apologize in advance if this has already been reported or corrected.

Check out version 2 and the relation members:
https://www.openstreetmap.org/relation/15963638/history

Relatify has added 11 sections of road to the restriction relation.

**Comment by Zaczero** [comment]
Yup, I know about that :slightly_frowning_face: The restriction relation support has to be improved.
